.github/workflows/conda-macos.yml

```diff
@@ -0,0 +1,36 @@
+name: conda-forge-macos
+on:
+  schedule:
+    # Run this workflow once a day
+    - cron: "0 0 * * *"
```

Comment:
Just a little nit pick, but can we schedule to weekly/biweekly instead?
Let's save some compute time and I don't think there are that many changes
happening in the build pipeline of pygraphviz?

@rossbar What do you think?

Reply:
Sounds good to me, I switched the cron scheduling to weekly for the conda-forge jobs.